### ISSUE TITLE
Error in plot_cells when plotting gene expression level

### ISSUE THREAD

**Issue by lixin4306ren** [cole-trapnell-lab/monocle3#125]
command I used:
```
plot_cells(cds, genes=c("CD34"))
```

error information:
```
No trajectory to plot. Has learn_graph() been called yet?
Error in monocle3::plot_cells(cds, genes = c("CD34")) : 
  trying to get slot "x" from an object of a basic class ("matrix") with no slots
```

I checked the code and found that it's caused by `cds_exprs@x = round(cds_exprs@x)`.

`cds_exprs` is a matrix which is generated by `SingleCellExperiment::counts(cds)` and there is no any slot within it. I think it's only for sparse matrix.

**Comment by hpliner** [comment]
Can you check the class of counts(cds) ? I think this may be related to #115 Which should be fixed in the 0.1.1 release being put out today

**Comment by hpliner** [comment]
I believe this was fixed, reopen if not!
